mkmetalink.py

```diff
@@ -53,7 +53,7 @@
         <file name="{{ file.path.name }}">
             <mm0:timestamp>{{ file.timestamp }}</mm0:timestamp>
             <size>{{ file.stat.st_size }}</size>
-            <resources>
+            <resources maxconnections="1">
```

Comment:
If that's about max parallel connections, do we really want to limit it to 1? DNF use more by default and it seems to make a difference.

Reply:
I took the value from fedora. I don't know what it means so I don't think I'd like to mess around it.